src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * This class represents a task.
+ * It contains the name of the task.
+ */
+package duke.tasks;
+public class Task {
+    private String name;
+    private boolean isDone;
+
+    /**
+     * Constructs a `Task` object with the specified name.
+     *
+     * @param name The name of the deadline.
+     */
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
```

Comment:
Love this naming  of isDone instead of poorer names like isStatus in the code of others